recipes/sphinx-jsonschema/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sphinx-jsonschema" %}
+{% set version = "1.16.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sphinx-jsonschema-{{ version }}.tar.gz
+  sha256: 5728b3ece1fe475c285d5ae16953242c51b4e6852f48f0bb734cd8408f3ac8b3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - docutils
```

Comment:
This really should have a dependency on sphinx since you need sphinx for this to function.

Reply:
I would have expected that the dependencies should be kept to the absolute bare minimum: `sphinx` isn't strictly a dependency of the `sphinx-jsonschema` package. But I suppose it does have the advantage that if one uninstalls sphinx, then `sphinx-jsonschema` would also be uninstalled. I'll add it in.